docs/upgrade/3.3.rst

```diff
@@ -16,6 +16,7 @@ by major code cleanup to remove compatibility shims.
 What's new in 3.3
 *****************
 
+* Improve compatibility with Django 1.9 esp. relating to the handling of `related_name`s of model fields in plugins.
```

Comment:
This should a be a double-backtick in RST: \`\`related_name\`\`.

Also, I have the feeling that the trailing ``s`` is going to break it...